### PR TITLE
fix(JS/RN): fixed code indent for JS/RN in Auth category

### PR DESCRIPTION
**Pull request by tannerabread**
#### Description of changes:
Fixed indent from 4-space to 2-space tabs in Auth category on JS & RN

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [x] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by abdallahshaban557 — DISMISSED** [review]
LGTM

File: src/fragments/lib/auth/common/device_features/common.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ amplify auth console
 **Select the following option** to open the Cognito User Pool console:
 ```bash
 ? Which Console
-    User Pool
+  User Pool
```
Comment (cshfang) [inline src/fragments/lib/auth/common/device_features/common.mdx:18]: Does this count as code indentation? I'm not sure what the updated guidelines are around CLI prompts

**Review by cshfang — APPROVED** [review]

File: src/fragments/lib/auth/common/device_features/common.mdx (hunk 1 above)
Comment (rachelnabors) [reply src/fragments/lib/auth/common/device_features/common.mdx:18]: CLI prompts (living in a terminal) are not the same as code blocks (which live in a code editor). Reflect the terminal. If things are not indented, don't indent. If they are, use the same spacing. 

Can you provide a screenshot of how this dialog looks in the terminal?

File: src/fragments/lib/auth/js/react-native-social.mdx [hunk 2]
```diff
@@ -83,23 +83,23 @@ For React Native applications, you need to define a custom URL scheme for your a
...
-             <key>CFBundleURLSchemes</key>
-             <array>
-                 <string>myapp</string>
-             </array>
-         </dict>
-     </array>
-
-     <!-- ... -->
-     </dict>
+  <dict>
```
Comment (cshfang) [inline src/fragments/lib/auth/js/react-native-social.mdx:86]: Just wanna double check if we have a separate standard for markup or if it falls under the same one

Comment (tannerabread) [reply src/fragments/lib/auth/js/react-native-social.mdx:86]: another good question for @rachelnabors

**Review by cshfang — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**